cypress/integration/product/create_bill_of_material.js

```diff
@@ -42,7 +39,7 @@ describe('Create Product', function() {
     cy.fixture('product/simple_product.json').then(productJson => {
       Object.assign(new Product(), productJson)
         .setName(productComponentName)
-        .setProductCategory(productCategoryValue + '_' + productCategoryName)
+        .setProductCategory(productCategoryName + '_' + productCategoryName)
         .setStocked(true)
```

Comment:
Because selecting the category is always done in the format `categoryName_categoryValue`. 

Gebinde:
![image](https://user-images.githubusercontent.com/4482210/63415073-cde92500-c405-11e9-8c80-c42273ca0b37.png)
![image](https://user-images.githubusercontent.com/4482210/63415098-de010480-c405-11e9-895a-10c81e776d33.png)

Or a test category:
![image](https://user-images.githubusercontent.com/4482210/63415142-ef4a1100-c405-11e9-8481-af693436c85f.png)
![image](https://user-images.githubusercontent.com/4482210/63415189-00931d80-c406-11e9-8492-79ed540b1e32.png)

Reply:
ok, so we have a convention, because I have tried without appending and it worked.